Dockerfile

```diff
@@ -27,3 +33,5 @@ RUN if [ "${BUILD_TYPE}" = "dev" ]; then \
     && pip install ${PIP_OPTIONS} jupyter notebook
 
 ENV PIP_OPTIONS ""
+
+COPY . .
```

Comment:
In this step, the dummy optuna get replaced with the true optuna.

Reply:
This trick might have some side effects that I haven't noticed yet though

Reply:
(This trick might have side effects that I haven't noticed yet though)

Reply:
We can verify this trick works by running an example script after the image build.